api/src/main/java/org/pmiops/workbench/config/RetryConfig.java

```diff
@@ -14,6 +18,7 @@
 import org.springframework.retry.policy.SimpleRetryPolicy;
 
 @Configuration
+@Import({NotebooksRetryHandler.class, GoogleRetryHandler.class, FirecloudRetryHandler.class})
```

Comment:
Hmmm... this is importing all of the subclasses of `RetryConfig.ResponseCodeRetryPolicy`. That seems a bit circular to me. What was the rationale for this change? Please also add a comment here explaining the criteria for adding another handler here.

Reply:
Aside: If the intent is to import a bunch of classes that have something in common (like a parent interface), I bet there's a pattern for that, such that another new subclass would automatically get picked up.

Reply:
Thanks, I find this to be much clearer.